Configure

```diff
@@ -2812,23 +2835,7 @@ or position independent code, please let us know (but please first make sure
 you have tried with a current version of OpenSSL).
 EOF
 
-print <<"EOF";
-
-**********************************************************************
-***                                                                ***
-***   OpenSSL has been successfully configured                     ***
-***                                                                ***
-***   If you encounter a problem while building, please open an    ***
-***   issue on GitHub <https://github.com/openssl/openssl/issues>  ***
-***   and include the output from the following command:           ***
-***                                                                ***
-***       perl configdata.pm --dump                                ***
-***                                                                ***
-***   (If you are new to OpenSSL, you might want to consult the    ***
-***   'Troubleshooting' section in the INSTALL.md file first)      ***
-***                                                                ***
-**********************************************************************
-EOF
+print $banner;
```

Comment:
A detail, you might want to consider this:

``` perl
$banner .= "\n" unless $banner =~ m|\R$|;
print $banner;
```

This is to make sure that, for example, `--banner=Blah-di-blah` doesn't give you a weird looking prompt on Unixly environments:

``` console
$ ./Configure --banner=Blah-di-blah
...
Blah-di-blah$ [blinking cursor here]
```